integration/tests/epochs/suite.go

```diff
@@ -632,3 +680,33 @@ func (s *Suite) assertNetworkHealthyAfterVNChange(ctx context.Context, _ templat
 	// head should now be at-least 20 blocks higher from when we started
 	require.True(s.T(), header.Height-bootstrapHead.Height >= 20, fmt.Sprintf("expected head.Height %d to be higher than head from the snapshot the node was bootstraped with bootstrapHead.Height %d.", header.Height, bootstrapHead.Height))
 }
+
+// assertNetworkHealthyAfterLNChange after an collection node is removed or added to the network
+// this func can be used to perform sanity.
+// 1. Submit transaction to network that will target the newly staked LN by making sure the reference block ID
+// is after the first epoch starts.
+// 2. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
+func (s *Suite) assertNetworkHealthyAfterLNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
+	bootstrapHead, err := rootSnapshot.Head()
+	require.NoError(s.T(), err)
+
+	header, err := s.client.GetLatestSealedBlockHeader(ctx)
+	require.NoError(s.T(), err)
+
+	// head should now be at-least 20 blocks higher from when we started
+	require.Truef(s.T(), header.Height-bootstrapHead.Height >= 20, "expected head.Height %d to be higher than head from the snapshot the node was bootstraped with bootstrapHead.Height %d.", header.Height, bootstrapHead.Height)
```

Comment:
This is telling us that the AN we're talking to has sealed 20 blocks above the starting block used by the LN. But I don't think it really tells us anything about the health of the LN. I think we could remove this?

Reply:
https://github.com/onflow/flow-go/pull/1819/commits/4436b1ea50cc954e3019dcdabb4bb3727dcf9263